test/test_pipeline/components/test_setup_traditional_classification.py

```diff
@@ -117,12 +117,13 @@ def test_classifier_fit_predict(self, classifier, dataset):
         # if the result is same as the one in results
         y_pred = classifier.predict(X, predict_proba=True)
         assert np.allclose(y_pred, results['val_preds'], atol=1e-04)
-
+        assert y_pred.shape[0] == y.shape[0]
         # Test if classifier can score and
         # the result is same as in results
         score = classifier.score(X, y)
         assert score == results['val_score']
-
+        # Test if score is greater than 0.8
+        assert score >= 0.8
         # delete folder created by catboost
         if isinstance(classifier, CatboostModel):
             shutil.rmtree('./catboost_info')
```

Comment:
BTW, this is a super important change. We do not have to do it now, but eventually. I am struggling with the automl autopytorch because of this. Files get created in the current directory. They should be created in backend. This is just an FYI that we will have to change this in the future.

Reply:
actually if this is not a lot of rouble, this seems like super easy to fix.

One has to just add:

https://catboost.ai/docs/search/?query=train_dir


Meaning, can you please here https://github.com/LMZimmer/Auto-PyTorch_refactor/blob/45901a53327a1e982bf8e1ae09d2f182d9348e01/autoPyTorch/pipeline/components/setup/traditional_ml/classifier_models/classifiers.py#L112 add this line:

https://github.com/franchuterivera/Auto-PyTorch/blob/31381aee6557ebe184f4db42e62300e81edb2983/autoPyTorch/components/baselines/baselines.py#L162

I THINK that should work? so we can remove line 129 and no file gets created at top level. 


This is super important because singularity in our cluster do no allow files to be created in a read only file system.